### ISSUE TITLE
Drag and drop target indicator does not erase

### ISSUE THREAD

**Issue by jivadevoe** [Perspx/PXListView#11]
I have a project that's using PXListView.  It also allows for drag and drop of individual cells.  The cells, when being dragged and dropped, in my case, cannot be dropped on another cell.  Only on the spaces between the cells.  So I have configured my delegate to return NO for PXListViewDropOn, and YES for the other drops, for the delegate method -listView:acceptDrop:row:dropHighlight:

This seems to work well, except that when I drag a cell down, and then bring it back up (without dropping it) the target indicator (the blue line that shows up between cells) persists on all cells I dragged over below the current cell.

So, in other words, I grab the top cell, and begin dragging down, I get a blue line between each set of other cells, as expected, and the blue line moves with my drag.  If I change directions, and begin dragging up, the blue line follows me, but is not erased from the lower cells, so instead it leaves a blue line between all the cells.

You can find a sample project that reproduces this here:

http://www.random-ideas.net/Dropbox/ListViewTest.zip
